README.md

```diff
@@ -55,370 +72,81 @@ breaking algorithm:
-[Exposed=Window,Worker]
-interface CanvasFormattedTextLine {
-  readonly attribute double height;
-  readonly attribute double width;
-  CanvasFormattedTextPosition? nextPosition();
-  readonly attribute FrozenArray<CanvasFormattedTextLineSegment> lineSegments;
-};
-
-
-dictionary CanvasFormattedTextLineSegment {
-  required CanvasFormattedTextPosition beginPosition;
-  required CanvasFormattedTextPosition endPosition;
-  required boolean isRTL;
-  required TextMetrics textMetrics;
-};
-```
+characteristics (e.g., size, bold, italic) are needed. 
+
+## Goals
+
+The browser already has a powerful line breaking, text shaping component used
+for regular HTML and SVG layout. Unfortunately, this component has been tightly coupled
+into the browser's rendering pipeline, and not exposed in an imperative way. 
+Furthermore HTML's DOM provides only a very limited set of metrics for text elements
+and almost no insight into the post-layout details of how the source text was
+formatted. SVG's DOM is slightly more helpful by providing some additional metrics.
+
+Our goal is to create an abstraction that allows authors to collect multi-line
+formatted text into a data model that is independent of the HTML/SVG data model
+but can still leverage the power of the browser's native line breaking and text
+shaping component. We think it is valuable to re-use formatting principles from CSS
+as much as possible. Given the data model, we also want to provide a way to
+initiate layout and rendering of that data model to a canvas, and likewise provide
+a way to initiate layout and then read-back the metrics that describe how the text
+was formatted and laid out given prescribed layout constraints (without requiring
+the text to be rendered at all).
+
+## Related Work
+
+This proposal builds on a variety of existing and proposed specifications in the 
+web platform, for whose efforts we are very grateful, and from whom we expect to
+get lots of feedback.
+* [CSS Text 3](https://drafts.csswg.org/css-text-3/)
+* [CSS Inline Layout 3](https://drafts.csswg.org/css-inline-3/)
+* [CSS Houdini Layout API](https://drafts.css-houdini.org/css-layout-api-1/)
+* [HTML Canvas API and TextMetrics](https://html.spec.whatwg.org/multipage/canvas.html#drawing-text-to-the-bitmap)
+* [SVG Text](https://svgwg.org/svg2-draft/text.html)
+* [Proposed Canvas Text Modifiers](https://github.com/fserb/canvas2D/blob/master/spec/text-modifiers.md)
+* [CSS Houdini Font Metrics API](https://drafts.css-houdini.org/font-metrics-api/)
+* [CSS Houdini Box Tree API](https://drafts.css-houdini.org/box-tree-api-1/)
+* [Other platforms for drawing formatted text](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/drawing-formatted-text)
+
+## Explainers
+
+We will be pursing consensus on each of the three aspects of the formatted text 
+problem space, starting with the data model and then detailing how it can be rendered
+and how it's metrics can be extracted:
```

Comment:
it's text metrics - so that people dont confuse this with some thing like perf metrics.

Reply:
Again good catch; I searched the whole Readme for "metrics" and made sure that "text metrics" was used in all the places it made sense.

Reply:
See https://github.com/WICG/canvas-formatted-text/commit/ee6113391b5048638e5c58d4d9da60f022481387